src/main/java/com/oltpbenchmark/benchmarks/templated/README.md

```diff
@@ -54,3 +55,47 @@ java -jar benchbase.jar -b templated -c config/sqlserver/sample_templated_config
 ```
 
 > For additional examples, please refer to the build pipeline definition in the [`maven.yml`](../../../../../../../.github/workflows/maven.yml#L423) Github Actions workflow file.
+
+## Value Distributions
+
+In order to support more variety in templated queries, it is possible to use a whole distribution of values instead of a single static value in a templated query
+
+Each time the query is run, a different value will substitute the `?`.
+
+```xml
+<templates>
+    <template name="MyTemplate">
+        <query><![CDATA[SELECT * FROM MyTable WHERE id = ?]]></query>
+        <types>
+            <type>INTEGER</type>
+        </types>
+        <values>
+            <value dist="uniform" min="0" max="1000" seed="1"/>
+        </values>
+         <values>
+            <value dist="zipf" max="1000" seed="1"/>
+        </values>
+    </template>
+    <!-- ... -->
+<templates>
+```
+
+The distributions are dependent on the type of the value. Currently, the following type-distributions pairs are supported:
+| Type | uniform | binomial | zipfian | scrambled zipfian |
+|---|:---:|:---:|:---:|:---:|
+| INTEGER | X | X | X | X |
+| FLOAT / REAL | X | X|- | - |
+| BIGINT | X | X | X | X |
+| VARCHAR / STRING | X | -| -| -|
+| TIMESTAMP | X | X |X |X |
+| DATE | X | X| X| X|
+| TIME | X |X | X| X|
```

Comment:
nit about the whitespace - aligning these column markers will make it easier to edit the markdown in the future